packages/core/integration-tests/test/css-modules.js

```diff
@@ -547,7 +547,7 @@ describe('css modules', () => {
       },
     ]);
   });
-  // Forked because experimental bundler will not merge bundles of same types if they do not share all their bundlegroups
```

Comment:
Removing this comment because this feature changes the behavior to match legacy bundler